packages/jumpstarter/jumpstarter/streams/progress.py

```diff
@@ -27,14 +109,22 @@ async def receive(self):
         item = await self.stream.receive()
 
         if self.__recv is None:
-            self.__recv = tqdm(desc="read", **TQDM_KWARGS)
+            if self.logging:
+                self.__recv = logging_tqdm(desc="read", **TQDM_KWARGS)
```

Comment:
```suggestion
                self.__recv = logging_tqdm(desc="transfer", **TQDM_KWARGS)
```

I have been thinking, what if we change this to transfer? , from the user perspective it's sometimes inverted.